src/predict.py

```diff
@@ -0,0 +1,114 @@
+"""
+Notes
+-----
+This module contains the functions for audiobook_gen that handle text-to-speech generation.
+The functions take in the preprocessed text and invoke the Silero package to generate audio tensors.
+"""
+
+__all__ = ['load_model', 'generate_audio', 'predict']
```

Comment:
You shouldn't need this. See other file for more comments.

Reply:
Removed `__all__`.